mlrun/projects/pipelines.py

```diff
@@ -1078,6 +1059,13 @@ def load_and_run(
     if load_only:
         return
 
+    # extract "start" notification if exists
+    start_notifications = [
+        mlrun.model.Notification.from_dict(notification)
+        for notification in context.to_dict().get("spec", {}).get("notifications", [])
```

Comment:
Would be much better to add an accessor to the notifications in the context object (it already has them in `self._notifications`). Probably was never done since nobody needed it so far...